Automattic/wp-calypso
# Tabbed plans page in signup flow

#### Changes proposed in this Pull Request
This PR continues the work from the proof of concept and implements @olaolusoga's designs posted at pdgrnI-5g-p2, and includes @victorespigares additional copy edits outlined at pdgrnI-87-p2.

Here are screenshots of the implementation for both viewports:


![screencapture-calypso-localhost-3000-start-plans-2021-09-17-10_41_24](https://user-images.githubusercontent.com/35781181/133801800-74bbb78a-6f39-4f53-8de3-ceca6af87803.png)

![screencapture-calypso-localhost-3000-start-plans-2021-09-17-10_09_42](https://user-images.githubusercontent.com/35781181/133801809-774bcc08-ceb7-4df1-8cfd-0ad529f2dd95.png)

#### Testing instructions



* Checkout this PR and start Calypso. Alternatively, you can use the Calypso Live link below.
* Add yourself to the treatment variation of the tabbed_plans_pocExplat test.
* Navigate to `/start/new` and go through the signup flow.
* Test the flow using a variety of plan and term-length combinations.
* Complete the signup flow, including purchase, for both free and paid plans to confirm the flow still works as expected.
* Repeat the process at both viewports (above and below 600px).



Related to #55768
